.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: csharp
+
+sudo: false  # use the new container-based Travis infrastructure
+os:
+  - osx
+  - linux
+install:
+  - curl -sS http://storage.bos.xamarin.com/bot-provisioning/PortableReferenceAssemblies-2014-04-14.zip > /tmp/pcl-assemblies.zip
+  - unzip /tmp/pcl-assemblies.zip -d /tmp/pcl-assemblies && mv /tmp/pcl-assemblies/PortableReferenceAssemblies-2014-04-14 /tmp/pcl-assemblies/.NETPortable
```

Comment:
I think you meant https://travis-ci.org/naveensrinivasan/octokit.net/jobs/96850533.

If you look at the actual build log, the error there is from something completely unrelated (`System.TypeInitializationException: The type initializer for 'System.Collections.Generic.List'1' threw an exception. ---> System.Threading.ThreadAbortException: ` during tests).